deploy/test/test_cases/TEST_ID_18_helm_multiple_provider_multiple_secrets.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that two Secrets Provider Jobs deploy successfully in the same namespace
+# TODO verify Job start up and completed logs
+
+setup_helm_environment
+
+echo "Create second secret"
+create_k8s_secret_for_helm_deployment "another"
+set_secret secrets/another_test_secret another-some-secret-value
```

Comment:
is this `set_conjur_secret`? if so, let's change the name to be more explicit

Reply:
you got it!

Reply:
choosing your battles, eh? :)